src/pages/home/ReportScreen.js

```diff
@@ -128,6 +131,16 @@ class ReportScreen extends React.Component {
     }
 
     componentDidMount() {
+        this.unsubscribeVisibilityListener = Visibility.onVisibilityChange(() => {
+            // If the report is not fully visible (AKA on small screen devices and LHR is open) or the report is optimistic (AKA not yet created)
+            // we don't need to call openReport again
```

Comment:
I don't think "again" fits here, since this is in `componentDidMount` right? Does the comment still make sense without "again"? Or can you explain where else we're calling openReport?

Reply:
Ok nice yeah that makes sense 👍  So ya please remove `again` and then this comment should be good to go 👍